builtin/logical/cassandra/path_roles.go

```diff
@@ -123,11 +128,14 @@ func (b *backend) pathRoleCreate(
 		return logical.ErrorResponse(fmt.Sprintf(
 			"Error parsing lease value of %s: %s", leaseRaw, err)), nil
 	}
+	
+	consistencyStr := data.Get("consistency").(string)	
```

Comment:
I was thinking the same thing, but I think we want to store it as a string. The consistency type is numerical so it wouldn't make much sense to users if they look at a role.